### DISCUSSION TITLE
Show all jobs of a workflow, including those depending on another job

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#17000] [Actions]
Right now when one job depends on the success/failure of another one is it hidden in the "Jobs" list when viewing active workflow runs.
Only after the required job succeeds/fails is the next job shown within that list, which feels kind of weird, because in the general overview are all jobs shown within the flowchart.

Because of this would I propose to show all jobs of a run, even those depending on another job to finish.
To indicate those jobs, could perhaps a line or arrow be used to indicate this.
Alternatively could the job have an indeed, similar to PR file-trees in the current feature preview.

This would make it a bit more clear what jobs would come after one has finished successfully or not.

**Comment by Salman-dott** [comment]


**Reply by Andre601** [reply]
Wrong type of "job"
